android/src/main/java/io/wazo/callkeep/RNCallKeepModule.java

```diff
@@ -49,7 +49,7 @@
 
 // @see https://github.com/kbagchiGWC/voice-quickstart-android/blob/9a2aff7fbe0d0a5ae9457b48e9ad408740dfb968/exampleConnectionService/src/main/java/com/twilio/voice/examples/connectionservice/VoiceConnectionServiceActivity.java
 public class RNCallKeepModule extends ReactContextBaseJavaModule {
-    public static final int REQUEST_READ_PHONE_STATE = 394858;
+    public static final int REQUEST_READ_PHONE_STATE = 1337;
     public static final int REQUEST_REGISTER_CALL_PROVIDER = 394859;
```

Comment:
Should we also lower this number for `REQUEST_REGISTER_CALL_PROVIDER ` ?

Reply:
I grep the codebase and can not find where REQUEST_REGISTER_CALL_PROVIDER is used, maybe just delete it?